src/ResourceManager/Network/Commands.Network/ApplicationGateway/NewAzureApplicationGatewayCommand.cs

```diff
@@ -143,6 +143,12 @@ public class NewAzureApplicationGatewayCommand : ApplicationGatewayBaseCmdlet
             HelpMessage = "Firewall configuration")]
         public virtual PSApplicationGatewayWebApplicationFirewallConfiguration WebApplicationFirewallConfiguration { get; set; }
 
+        [Parameter(
+            Mandatory = false,
+            ValueFromPipelineByPropertyName = true,
```

Comment:
@akshaysngupta switch parameters should not have the `ValueFromPipelineByPropertyName` property

Reply:
@akshaysngupta please follow up on this comment.